tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = py27,py34,py35,py36,pypy
+
+[testenv]
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+deps =
+    --no-binary
+     zc.recipe.egg
```

Comment:
Hmm, I didn't realize `--no-binary` took an `=`. The help description (`--no-binary <format_control>`) doesn't imply that.

I'm not sure I like the "repetition" in the latter, but I'll play with this.